app/config/Module.scala

```diff
@@ -17,7 +17,7 @@
 package config
 
 import com.google.inject.AbstractModule
-import controllers.actions.{AuthAction, AuthActionImpl, DataRetrievalAction, DataRetrievalActionImpl, MovementAction, MovementActionImpl}
+import controllers.actions.{AuthAction, AuthActionImpl, DataRequiredAction, DataRequiredActionImpl, DataRetrievalAction, DataRetrievalActionImpl, MovementAction, MovementActionImpl, UserAllowListAction, UserAllowListActionImpl}
```

Comment:
```suggestion
import controllers.actions._
```
Might as well just import it all at this point (that's what IntelliJ suggests anyway)

Reply:
Updated